dep/CLI11/CLI11.hpp

```diff
@@ -913,11 +913,11 @@ public:
     {
         CLI11_ERROR_DEF(ParseError, ArgumentMismatch)
         CLI11_ERROR_SIMPLE(ArgumentMismatch)
-        ArgumentMismatch(std::string name, int expected, size_t recieved) :
+        ArgumentMismatch(std::string name, int expected, size_t received) :
             ArgumentMismatch(expected > 0 ? ("Expected exactly " + std::to_string(expected) + " arguments to " + name +
-                                             ", got " + std::to_string(recieved)) :
+                                             ", got " + std::to_string(received)) :
                                             ("Expected at least " + std::to_string(-expected) + " arguments to " + name +
-                                             ", got " + std::to_string(recieved)),
+                                             ", got " + std::to_string(received)),

```

Comment:
Don't touch things in the ./dep/ folder, please. We need to submit this to be fixed upstream at https://github.com/CLIUtils/CLI11

Reply:
Not sure I'll have time to run my tooling this week. Might look on Sunday.

Dropped. (and I cloned the upstream repo.)

Reply:
CLIUtils/CLI11#429

Reply:
Wow they merged that fast. I've now made #4546 to update our copy of CLIUtils. Thanks!

Reply:
Yeah, I was really impressed by that.